File: test/fixtures/Sample.swift
import Foundation
import UIKit

class Inventory {
    var name: String
    init(name: String) { self.name = name }
    func reserve(sku: String, qty: Int) -> Bool { return qty > 0 }
    private func helper() {}
}

protocol Reader {
    func read() -> Int
}

struct Point {
    let x: Int
}

func topLevel(x: Int) -> Int { return x * 2 }
let MAX = 100

File: test/fixtures/Sample.kt
package com.example

import kotlin.collections.List
import kotlin.io.println

class Inventory(val name: String, var count: Int) {
    fun reserve(sku: String, qty: Int): Boolean = qty > 0
    private fun helper() {}
}

object Constants {
    const val MAX = 100
}

fun topLevel(x: Int): Int = x * 2
